lib/dgram.js

```diff
@@ -121,6 +122,12 @@ function Socket(type, listener) {
 
   // If true - UV_UDP_REUSEADDR flag will be set
   this._reuseAddr = options && options.reuseAddr;
+  
+  if (options && options.recvBufferSize)
```

Comment:
They are used in `startListening()` which is called after the socket has been bound. They will in turn determine if `setRecvBufferSize()` and `setSendBufferSize()` are called if the specific options have been set against the `Socket` created with `dgram.createSocket()`. Otherwise `setRecvBufferSize()` and `setSendBufferSize()` can be called on the `Socket` object.